tools/js-tools/package.json

```diff
@@ -26,11 +29,12 @@
 		"eslint-plugin-es": "4.1.0",
 		"eslint-plugin-es5": "1.5.0",
 		"eslint-plugin-inclusive-language": "2.2.0",
+		"eslint-plugin-import": "2.25.4",
 		"eslint-plugin-jest": "26.0.0",
 		"eslint-plugin-jsdoc": "37.7.0",
 		"eslint-plugin-jsx-a11y": "6.5.1",
 		"eslint-plugin-lodash": "7.3.0",
-		"eslint-plugin-prettier": "4.0.0",
+		"eslint-plugin-prettier": "3.4.1",
```

Comment:
Why is this being downgraded?

Reply:
There was a conflict between different prettier versions and thus ESLint threw this error

`ESLint couldn't determine the plugin "prettier" uniquely.`

Reply:
Here is the exact error message
<img width="1098" alt="Screenshot 2022-03-24 at 9 42 41 AM" src="https://user-images.githubusercontent.com/18226415/159840779-e5ba3194-00c1-4dc7-bed5-cc40d7bd0a33.png">

Reply:
Ugh. That's going to be annoying if we have to sync various random plugin versions with whatever `@wordpress/eslint-plugin` happens to include.

I'm tempted to recommend using `.pnpmfile.cjs` to turn `@wordpress/eslint-plugin`'s eslint plugin deps into peer deps instead. That might look like
```diff
diff --git a/.pnpmfile.cjs b/.pnpmfile.cjs
--- a/.pnpmfile.cjs
+++ b/.pnpmfile.cjs
@@ -52,6 +52,16 @@ function fixDeps( pkg ) {
                pkg.dependencies[ 'jest-environment-jsdom' ] = '^27';
        }
 
+       // Turn @wordpress/eslint-plugin's eslint plugin deps into peer deps.
+       if ( pkg.name === '@wordpress/eslint-plugin' ) {
+               for ( const [ dep, ver ] of Object.entries( pkg.dependencies ) ) {
+                       if ( dep.startsWith( 'eslint-plugin-' ) || dep.endsWith( '/eslint-plugin' ) ) {
+                               delete pkg.dependencies[ dep ];
+                               pkg.peerDependencies[ dep ] = ver.replace( /^\^?/, '>=' );
+                       }
+               }
+       }
+
        // Unpin browserslist here.
        if (
                pkg.name === 'react-dev-utils' &&
```

Reply:
I have added the suggestion to `.pnpmfile.cjs`